### PR TITLE
Publication section added

### PR DESCRIPTION
**Pull request by Aarontama**

### CODE REVIEW

File: users/tamashia/index.md [hunk 1]
```diff
@@ -54,6 +54,11 @@ Under Construction 8/4/17
 
 ***
 
+### Publications
+1. R.O. Hughes, J.T. Burke, R.J. Casperson, S. Ota, 1, S. Fisher, J. Parker, C.W. Beausang, M. Dag, P. Humby, J. Koglin, E. McCleskey, A.B. McIntosh, A. Saastamoinen, A.S. Tamashiro, E. Wilson, T.C. Wu, "The hyperion particle-&gamma detector array", *Nuclear Instruments and Methods in Physics Research Section A: Accelerators, Spectrometers, Detectors and Associated Equipment*, **856**, 47-52 (2017).
```
Comment (albeanth) [inline users/tamashia/index.md]: quick question - what is the "&" for in your publication title?

Comment (Aarontama) [reply users/tamashia/index.md]: trying to use special characters, so I want the greek gamma symbol where the &gamma is.

Comment (Aarontama) [reply users/tamashia/index.md]: hold on, i need to change it to \gamma

File: users/tamashia/index.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Under Construction 8/4/17
 ***
 
 ### Publications
-1. R.O. Hughes, J.T. Burke, R.J. Casperson, S. Ota, 1, S. Fisher, J. Parker, C.W. Beausang, M. Dag, P. Humby, J. Koglin, E. McCleskey, A.B. McIntosh, A. Saastamoinen, A.S. Tamashiro, E. Wilson, T.C. Wu, "The hyperion particle-&gamma detector array", *Nuclear Instruments and Methods in Physics Research Section A: Accelerators, Spectrometers, Detectors and Associated Equipment*, **856**, 47-52 (2017).
+1. R.O. Hughes, J.T. Burke, R.J. Casperson, S. Ota, 1, S. Fisher, J. Parker, C.W. Beausang, M. Dag, P. Humby, J. Koglin, E. McCleskey, A.B. McIntosh, A. Saastamoinen, A.S. Tamashiro, E. Wilson, T.C. Wu, "The hyperion particle-\gamma detector array", *Nuclear Instruments and Methods in Physics Research Section A: Accelerators, Spectrometers, Detectors and Associated Equipment*, **856**, 47-52 (2017).
```
Comment (albeanth) [inline users/tamashia/index.md]: @Aarontama if you want to use tex stuff you will need to put it in between $$ <insert> $$

At least that _should_ make it work... I've had troubles with this in the past... @rtrp/admins

Comment (harterj) [reply users/tamashia/index.md]: @Aarontama @albeanth 

Inline TeX is $$ *argument* $$

**Comment by albeanth** [comment]
@Aarontama can you update this so we can close this PR? 

Thanks man!

**Comment by Aarontama** [comment]
I believe I just updated my index.md using the $$ argument $$

**Comment by albeanth** [comment]
@rtrp/admins 🚢  it

**Comment by harterj** [comment]
Looks good. :shipit:

**Pull request merged**